gulp-tasks/utils/external-helpers.js

```diff
@@ -2,35 +2,36 @@
 // This is extracted from the Babel runtime (original source: https://github.com/babel/babel/blob/9e0f5235b1ca5167c368a576ad7c5af62d20b0e3/packages/babel-helpers/src/helpers.js#L240).
 // As part of the Rollup bundling process, it's injected once into workbox-core
 // and reused throughout all of the other modules, avoiding code duplication.
+// It's exposed as self.Z as a brute-force minification technique, and rollup
+// replaces references to babelHelpers.asyncToGenerator with self.Z as part of
+// the bundling process.
 // See https://github.com/GoogleChrome/workbox/pull/1048#issuecomment-344698046
 // for further background.
-self.babelHelpers = {
-  asyncToGenerator: function(fn) {
-    return function() {
-      var gen = fn.apply(this, arguments);
-      return new Promise(function(resolve, reject) {
-        function step(key, arg) {
-          try {
-            var info = gen[key](arg);
-            var value = info.value;
-          } catch (error) {
-            reject(error);
-            return;
-          }
+self.Z = function(fn) {
```

Comment:
Two things:

1.) Can we drop `function` for arrows (should be smaller).
2.) Should this be workbox.Z (or something to that affect. Could be workbox._). Main thinking is that self.Z is still user space.

Comparing:
```
self.Z
self._wb
self.workbox._
self.babelHelpers.asyncToGenerator
```